taipy/core/data/data_node.py

```diff
@@ -14,14 +14,15 @@
 import uuid
 from abc import abstractmethod
 from datetime import datetime, timedelta
-from typing import Any, Dict, List, Optional, Set, Tuple, Union
+from typing import Any, Dict, List, Optional, Set, Tuple, Union, cast
 
 import networkx as nx
 
 from taipy.common.config.common._validate_id import _validate_id
 from taipy.common.config.common.scope import Scope
 from taipy.common.logger._taipy_logger import _TaipyLogger
 
+from ... import Config
```

Comment:
This should be `from taipy.common.config import Config`, not the relative import.

Reply:
Ok but why?

Reply:
Oh I get it ... different packages, you're right it's cleaner